src/components/base/containers/Tooltip.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component, PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+import Overlay from 'react-overlays/lib/Overlay';
+import Transition from 'react-overlays/lib/Transition';
+import styled, { injectGlobal } from 'styled-components';
+import { colors } from '../../theme';
+
+injectGlobal(`
```

Comment:
I wonder if you could do something like this:

````javascript
const StyledTransition = styled(Transition)`
  opacity: ${props => props.visible ? 0.9 : 0;
  transition: opacity 200ms linear;
`;
````

Then just use StyledTransition in place of Transition and pass whatever you name the prop along. Something like

````jsx
function Fade(props) {
  return (
    <StyledTransition visible={props.in} {...props} />;
  );
}
```